docs/setup.md

```diff
@@ -29,6 +29,106 @@ When running the Oasis CLI for the first time, it will generate a configuration
 file and populate it with the current Mainnet and Testnet networks. It will also
 configure all [ParaTimes supported by the Oasis Foundation][paratimes].
 
+## Installing or Updating Oasis CLI on macOS
```

Comment:
Nice! Would you be up to making a tab widget for all platforms install instructions? The linux one should be equal to macOS, just without the gatekeeper step. The windows instructions should extract the zip file and put it inside the path. "Run the Oasis CLI by typing" part is then below the tab widget and should be common to all platforms.

For the example on how to use tab widgets in Docusaurus, you can check:
https://docs.oasis.io/build/rofl/prerequisites#conservative-setup-everything-docker

Reply:
Sure